ppr-ui/src/utils/validation-helper.ts

```diff
@@ -57,31 +57,31 @@ export function validateSearchAction (
     case APISearchTypes.INDIVIDUAL_DEBTOR:
       if (!first) {
         validation.searchValue.messageFirst = 'Enter a first name'
-      } else if (first?.length > 50) {
-        validation.searchValue.messageFirst = 'Maximum 50 characters'
+      } else if (first?.length > 15) {
+        validation.searchValue.messageFirst = 'Maximum 15 characters'
```

Comment:
Just to clarify, the task was to update PPR validations as well? I believe Debtor name is a PPR field.
Just double checkin :)

Reply:
Yes individual debtors are PPR parties, and the first, middle, last names can each be 50 characters.

Reply:
Good catch, thanks! Let me fix it.

Reply:
Reverted back to 50.